shoper.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pandas as pd
+from operator import itemgetter
+from re import search
+
+
+def deactivate_products(database, dc_price, replacement_price):
```

Comment:
Jak to rozumiem:
w jakiś sposób zmień reprezentację danego produktu (z daną ceną?) na taką, żeby wskazywała, że jest wyłączona.    
Jednak jest jeszcze jakaś wartość ```replacement_price``` o której nie ma nic w docstringu oraz nazwa funkcji wskazuje, na prosty switch 0/1 (z tego co widzę co się też dzieje w kodzie), a nie jeszcze zmianę cen.